controllers/glance_common.go

```diff
@@ -300,3 +301,26 @@ func GetServiceLabels(
 		common.OwnerSelector:     instance.Name,
 	}
 }
+
+// GetWSGIAnnotation - returns in the form of a boolean the value associated to
+// the "wsgi" annotation. It determines whether Glance will be deployed in WSGI
+// or ProxyPass mode.
+func GetWSGIAnnotation(
```

Comment:
might be good to add a generic one in https://github.com/openstack-k8s-operators/lib-common/blob/main/modules/common/annotations/network.go , which has parameters for
* the annotations map
* the key to check
* the expected default when not found

Reply:
I was thinking about this at the beginning, it might be a good idea. Let me draft something and I will come up with an improved approach for this.

Reply:
Done https://github.com/openstack-k8s-operators/lib-common/pull/628